source/testUtilities/mockComponentData.js

```diff
@@ -81,4 +81,16 @@ const mockCouldDos = [
   },
 ]
 
-export { mockGlobalState, mockProjects, mockCouldDos }
+const createDeepClone = ( object ) => {
+  let clone = {}
+  for ( let i in object ) {
+    if ( typeof object[i] === 'object' ) {
+      clone[i] = createDeepClone( object[i] )
+    } else {
+      clone[i] = object[i]
+    }
+  }
```

Comment:
What about if there is an array in the object? Lets make it work for an array as well.